### ISSUE TITLE
Create regular Patch Notes modal

### ISSUE THREAD

**Issue by julesfll** [thecourseforum/theCourseForum2#458]
**Is your feature request related to a problem? Please describe.**
- We make changes without informing users what is updated
- Having public updates shows we are actively working on the site
- Welcome to tcf modal shows up every time
- modal only shows on landing page

**Describe the solution you'd like**
- Patch notes update modal should show up
  - [ ] After a new patch is deployed
  - [ ] Only the first time the user visits after the patch
  - [ ] On whatever page the user first visits (as opposed to only landing page)
- [ ]  Have a documented procedure for updating the modal content

**Additional context**

**Comment by Bartleby2718** [comment]
Does this supersede #382?

**Comment by Bartleby2718** [comment]
Alternative to consider: instead of using modals, allow non-devs to write announcements on the admin page and make this a subtask of #415.

**Comment by borjainglefernandez** [comment]
> Alternative to consider: instead of using modals, allow non-devs to write announcements on the admin page and make this a subtask of #415.

Yes I like this idea much better. I looked into using modals when a user logs into the site the first time after a patch and we would have to store information about when the user last logged in.

**Comment by julesfll** [comment]
Notifications sounds good!
